templates/scenarios/csharp/notification-timer-trigger/teamsfx/app.local.yml.tpl

```diff
@@ -0,0 +1,50 @@
+version: 1.0.0
```

Comment:
There is already `app.local.yml`. Rename it instead of adding a new one.

Reply:
Yep, this is a rename pr. Git mistake this for a add operation.